#DoCoMo Dialogue API Request message
Header header
string utt 
string context 
string nickname
string nickname_y
string sex
string bloodtype
int16 birthdateY
int16 birthdateM
int16 birthdateD
int16 age
string constellations
string place
string mode
string t
